parcels/interaction/interactionkernel.py

```diff
@@ -125,16 +125,6 @@ def merge(self, kernel, kclass):
         return kclass(self._fieldset, self._ptype, pyfunc=pyfunc,
                       funcname=funcname)
 
-    def __add__(self, kernel):
-        if not isinstance(kernel, InteractionKernel):
-            kernel = InteractionKernel(self.fieldset, self.ptype, pyfunc=kernel)
-        return self.merge(kernel, InteractionKernel)
-
-    def __radd__(self, kernel):
-        if not isinstance(kernel, InteractionKernel):
-            kernel = InteractionKernel(self.fieldset, self.ptype, pyfunc=kernel)
-        return kernel.merge(self, InteractionKernel)
-
```

Comment:
This code was unnecessary as it is already covered by the BaseKernel dunder methods (which this is a subclass of)
```python
def __add__(self, kernel):
    if not isinstance(kernel, type(self)):
        kernel = type(self)(self.fieldset, self.ptype, pyfunc=kernel)
    return self.merge(kernel, type(self))

def __radd__(self, kernel):
    if not isinstance(kernel, type(self)):
        kernel = type(self)(self.fieldset, self.ptype, pyfunc=kernel)
    return kernel.merge(self, type(self))

```

Here `type(self)` would return `InteractionKernel` making these equivalent

Reply:
I've decided to leave this change for later